scripts/tests/scripts_regression_tests.py

```diff
@@ -1317,6 +1317,9 @@ def test_run_restart(self):
     ###########################################################################
     def test_run_restart_too_many_fails(self):
     ###########################################################################
+	if self._machine == "athena":
```

Comment:
This line is here for mistake, I implemented it to avoid this check, which was not working some times. In the "test_cime_case_test_walltime_mgmt_6" test, I introduced "athena" option because our machine uses only HH:MM values in JOB_WALLCLOCK_TIME and not HH:MM:SS.